games-strategy/0ad/0ad-0.0.22.recipe

```diff
@@ -0,0 +1,76 @@
+SUMMARY="O A.D. - Game"
+DESCRIPTION="A free, open-source, Real-Time Strategy (RTS) game."
+HOMEPAGE="https://play0ad.com"
+COPYRIGHT="Wildfire Games"
+LICENSE="0ADLicense"
+REVISION="1"
+SOURCE_URI="https://releases.wildfiregames.com/0ad-0.0.22-alpha-unix-build.tar.gz"
+SOURCE_DIR="0ad-0.0.22-alpha"
+CHECKSUM_SHA256="b50ef5aa8004bc60c542493d2c3ceff59ebdefadcd901d529c492c91503ed047"
+ARCHITECTURES="!x86_gcc2 ?x86_64 ?x86"
+SECONDARY_ARCHITECTURES="x86"
+PATCHES="0ad-0.0.22-alpha.patch"
+
+PROVIDES="
+		0ad$secondaryArchSuffix = $portVersion
+		app:0ad$secondaryArchSuffix = $portVersion
+"
+REQUIRES="
+		haiku$secondaryArchSuffix
+		lib:libSDL2$secondaryArchSuffix
+		lib:libvorbis$secondaryArchSuffix
+		lib:libpng$secondaryArchSuffix
+		lib:libcurl$secondaryArchSuffix
+		lib:libzlib$secondaryArchSuffix
+		lib:libxml2$secondaryArchSuffix
+		lib:libogg$secondaryArchSuffix
+		lib:libnspr4$secondaryArchSuffix
+		lib:openal$secondaryArchSuffix
+		0ad_data$secondaryArchSuffix
+"
+
+BUILD_REQUIRES="
+		boost${secondaryArchSuffix}_devel
+		nspr${secondaryArchSuffix}_devel
+		libxml2${secondaryArchSuffix}_devel
+"
+
+BUILD_PREREQUIRES="
+		haiku${secondaryArchSuffix}_devel
+		cmd:make
+		cmd:gcc$secondaryArchSuffix
+		cmd:g++$secondaryArchSuffix
+		cmd:ld$secondaryArchSuffix
+		cmd:pkg_config
+		cmd:perl
+		cmd:patch
+		cmd:python
+		boost_x86_devel
+"
+
+BUILD()
+{
+	echo `python --version`
+	export OriginalDir=`pwd`
+	cd "build/workspaces/"
+	fixPkgconfig
```

Comment:
Almost entirely sure this not needed, except if your package contains *.PC Files. But nevertheless mistakenly placed as it needs to be in Install() after "make install"